index.html

```diff
@@ -53,12 +65,16 @@ <h1>I’m Fadahunsi Glad to see you!</h1>
                 </a>
             </li>
             </ul>
+          </div>
         </div>
     </section>
     <main>
         <section id="myWork" class="my-works">
```

Comment:
- [ ] Please in this section, make sure the page is divided in half contains. The project's picture and the other half contains the project's description, these switch places with each project (picture is on the left and description on the right for the first project, description on the left and picture on the right for the second project and so on and so forth).
- See Original Design. 
![1](https://user-images.githubusercontent.com/63972670/171750595-34b776f0-b018-49f0-a754-ecbcebe9f0ea.PNG)
- See Your design

![2](https://user-images.githubusercontent.com/63972670/171751082-909c12a0-0913-46e7-a15d-fa21b2457d4e.PNG)